3512-0/about_dice_project.rb

```diff
@@ -0,0 +1,70 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+# class DiceSet
+#   code ...
+# end
+
+class DiceSet
+  attr_accessor :values
+
+  def roll(number)
+    @values = (1..number).to_a.shuffle
```

Comment:
Rly? 😞 Just tell me what would you do if I change `roll(5)` to `roll(1)` here 
( https://github.com/ElenaParchevskaya/summer-2019/blob/699b070c76bd6eccedbabd428503b54a630bb165/3512-0/about_dice_project.rb#L36 ) Rewrite